tests/dummy/app/templates/application.hbs

```diff
@@ -1,3 +1,34 @@
+<div class="fx-row fx-gap-px-10 margin-md fx-xalign-center">
+  <OSS::Avatar @image="https://reachr-assets.s3.us-west-2.amazonaws.com/influencer-server/influencer/7219681.png"
+   @initials="Ts" />
+  <OSS::Avatar @image="https://reachr-assets.s3.us-west-2.amazonaws.com/influencer-server/influencer/7219681.png"
+   @initials="Ts" @size='xs'/>
+  <OSS::Avatar @image="https://reachr-assets.s3.us-west-2.amazonaws.com/influencer-server/influencer/7219681.png"
+   @initials="Ts" @size='sm'/>
+  <OSS::Avatar @image="https://reachr-assets.s3.us-west-2.amazonaws.com/influencer-server/influencer/7219681.png"
+   @initials="Ts" @size='md'/>
+  <OSS::Avatar @image="https://reachr-assets.s3.us-west-2.amazonaws.com/influencer-server/influencer/7219681.png"
+   @initials="Ts" @size='lg'/>
+   <OSS::Avatar @image="https://reachr-assets.s3.us-wes-2.amazonaws.com/influencer-server/influencer/7219681.png"
+   @initials="Ts" @size='md'/>
+</div>
+
+<div class="fx-row fx-gap-px-10 margin-md fx-xalign-center">
+  <OSS::Avatar @initials="MI" />
+  <OSS::Avatar @initials="MI" @size='xs'/>
+  <OSS::Avatar @initials="MI" @size='sm'/>
+  <OSS::Avatar @initials="MI" @size='md'/>
+  <OSS::Avatar @initials="MI" @size='lg'/>
+</div>
+
+<div class="fx-row fx-gap-px-10 margin-md fx-xalign-center">
+  <OSS::Avatar />
+  <OSS::Avatar @size='xs'/>
+  <OSS::Avatar @size='sm'/>
+  <OSS::Avatar @size='md'/>
+  <OSS::Avatar @size='lg'/>
+</div>
+
```

Comment:
I know this is the local test app, but you should still use double quotes instead of single quotes